lib/hs/src/Thrift/Transport/Handle.hs

```diff
@@ -62,17 +63,12 @@ instance Transport Handle where
     tWrite = LBS.hPut
     tFlush = hFlush
 
-
--- | Type class for all types that can open a Handle. This class is used to
--- replace tOpen in the Transport type class.
-class HandleSource s where
```

Comment:
This class does not look really useful. I have removed it, but kept a `hOpenSocket` function to make the common use case of connecting to a socket concise.